components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.websocket.local/src/main/java/org/wso2/carbon/event/output/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java

```diff
@@ -20,12 +20,23 @@
 
 import org.wso2.carbon.event.output.adapter.core.*;
 import org.wso2.carbon.event.output.adapter.websocket.local.internal.util.WebsocketLocalEventAdapterConstants;
+import org.wso2.carbon.utils.CarbonUtils;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
 public class WebsocketLocalEventAdapterFactory extends OutputEventAdapterFactory {
```

Comment:
@lafernando 
Thank you for pointing out. Fixed in PR: https://github.com/wso2/carbon-analytics-common/pull/146